_posts/2019-10-4-manual-testing-how-to-become-a-better-tester-of-your-own-code.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title:  "Manual Testing: How to become a better tester of your own code"
+date: 2019-10-4 11:00:00
+categories: ["software-development", "software-quality"]
+author: abizzinotto
+---
+
+Manual testing is a necessary part of software development and quality assurance. And although it's important to have a dedicated tester in your team, you as a developer can also help speed up QA, and thus the software development process, by becoming a better manual tester of your own code.
+
+But how to do that? I'll cover 4 simple points that will help you get there!
+
+<!--more-->
+
+Manual testing is a necessary part of software development and quality assurance. It’s true that it is prone to errors, after all, every human activity is, and can also be time-consuming and boring. It’s also true that automated testing reduces testing time a great deal and is more reliable. But automated tests can only go so far. Human intuition and curiosity cannot be replaced by an automated task.
+
+Therefore, it is important to have a software tester in your team who knows not only the application but also the end user well. This person will be the “official” manual tester and will find those edge cases for you. But, as a developer, it is also important for you to stay on top of things and manually test your own code before moving on to the Quality Assurance phase. This way, you can ensure your code does exactly what it was intended to do.
+
+Here at [OmbuLabs](https://www.ombulabs.com) we follow some principles and guidelines to manually test our own code before it goes to QA. In this post, we’ll walk you through these guidelines.
+
+##It all starts with the user story
+
+Here at OmbuLabs we work with user stories. And every user story has a definition and a scope. Therefore, we have a bug that needs to be fixed, a feature that needs to be added, a chore that needs to be performed, and other situations specific to the project. What all of these have in common is they present a problem to be solved. As such, there is a specific result or outcome that is expected. 
+
+Now, we have automated tests in place to make sure a feature does what it needs and produces the expected outcome. But there are aspects that need manual testing. How does it look on the screen? What if the user decides to follow an alternate path to get the same result?
```

Comment:
`we have automated tests in place to make sure a feature does what it needs and produces the expected outcome.`

We also have automated tests for  "bug fixes"

Reply:
Feedback addressed and change made here https://github.com/ombulabs/blog/pull/204.